examples/whiteboard/public/main.js

```diff
@@ -16,6 +16,12 @@
   canvas.addEventListener('mouseup', onMouseUp, false);
   canvas.addEventListener('mouseout', onMouseUp, false);
   canvas.addEventListener('mousemove', throttle(onMouseMove, 10), false);
+  
+  //Touch support for mobile devices
+  canvas.addEventListener('touchstart', onMouseDown, false);
+  canvas.addEventListener('touchend', onMouseUp, false);
+  canvas.addEventListener('touchcancle', onMouseUp, false);
```

Comment:
this should be `touchcancel` rather than `touchcancl`

https://developer.mozilla.org/en-US/docs/Web/API/Touch_events#Setting_up_the_event_handlers

Reply:
Thanks for pointing that out :+1: 
Checkout the new commit, typo is fixed